src/node_contextify.cc

```diff
@@ -1632,6 +1632,12 @@ static std::vector<std::string_view> throws_only_in_cjs_error_messages = {
     "await is only valid in async functions and "
     "the top level bodies of modules"};
 
+static std::vector<std::string_view> maybe_top_level_await_errors = {
```

Comment:
Fyi https://github.com/nodejs/node/pull/58580 will likely land first and you will probably want to align this bit and/or be prepared for a merge conflict